tests/plugins/pytest_dut_monitor.py

```diff
@@ -0,0 +1,362 @@
+import pytest
+import paramiko
+import threading
+import logging
+import pprint
+import time
+import os
+import yaml
+
+from collections import OrderedDict
+from  datetime import datetime
+
+logger = logging.getLogger(__name__)
+DUT_MONITOR = "/tmp/dut_monitor.py"
+DUT_CPU_LOG = "/tmp/cpu.log"
+DUT_RAM_LOG = "/tmp/ram.log"
+DUT_HDD_LOG = "/tmp/hdd.log"
+THRESHOLDS = os.path.join(os.path.split(__file__)[0], "thresholds.yml")
```

Comment:
Better to let user specify different threasholds file, what do you think? It's kind of change to let users of tests disintegrate their topology/DUT configurations from test logic

Reply:
Added possibility to specify thresholds file via "--thresholds_file" pytest option.